src/domains/job/main/components/JobPostList.jsx

```
@@ -1,14 +1,16 @@
 import WorkBoard from '../../../../components/web/WorkBoard'
 import MobileWorkBoard from '../../../../components/app/MobileWorkBoard'
 import useAuthStore from '../../../../store/login/useAuthStore'
+import getExperienceLabel from '../../common/utils/getExperienceLabel'
+
 const JobPostList = ({ posts, navigate }) => {
```

ESLint

[error] 'posts' is missing in props validation

(react/prop-types)